### DISCUSSION TITLE
选择Obsidian作为md文件默认打开软件后打开md文件显示新建库问题

### DISCUSSION THREAD

**Discussion by peisencn** [obsidianzh/forum#120] [Q&A常见问题]
选择Obsidian作为md文件默认打开软件后打开md文件会显示新建库或选择已有库
选择完已有库后会进入默认界面，并没有打开想打开的文件
请问有没有方法可以设置为双击md文件，默认在Obsidian库里打开这个文件？
如能解答，不胜感激

**Comment by grittang** [comment]
我会建议你**直接把 Markdown 文档拷贝到 Obsidian 仓库中对应的位置**，Obsidian 会自动识别。

和一般软件不一样，目前 Obsidian 没有双击打开 MD 文档的方式。你可以试试在 Obsidian 中右键 “Copy Obsidian URL”, 比如我这里 Copy 后得到的链接形式是：obsidian://open?vault=PKM&file=~%20AS2PKM%20MOC

于是可以写一段 Python，把这个链接中的 Obsidian 协议写进去，再把这个 Python 文件的路径添加到注册表。总之比较麻烦。

**Comment by amitbha** [comment]
可以在官方论坛提新特性需求
不知道插件能否实现
